x/concentrated-liquidity/math/tick.go

```diff
@@ -120,55 +120,59 @@ func PriceToTick(price sdk.Dec) (sdk.Int, error) {
 
 // PriceToTickRoundDown takes a price and returns the corresponding tick index.
 // If tickSpacing is provided, the tick index will be rounded down to the nearest multiple of tickSpacing.
-func PriceToTickRoundDown(price sdk.Dec, tickSpacing uint64) (sdk.Int, error) {
+func PriceToTickRoundDown(price sdk.Dec, tickSpacing uint64) (int64, error) {
```

Comment:
Should we add a comment or make an error if `tickSpacking >= (1 >> 63)`

Reply:
@adam's comment is the reason why I did not add checks. I don't see a world where tick spacing would be this big. I will add a comment to the godoc for clarity in the future

Reply:
[8a6a301](https://github.com/osmosis-labs/osmosis/pull/5189/commits/8a6a3015ee30af2e9fa0d77749cd6df5ab31f23d)